site/docs/v1/tech/apis/_ip-acl-response-body-base.adoc

```diff
@@ -0,0 +1,37 @@
+==== Response Body
```

Comment:
This is already added by the user of this include, so it ends up twice on the page

Reply:
Fixed.